app/chat/page.tsx

```diff
@@ -0,0 +1,108 @@
+"use client";
+
+import { useState } from "react";
+import { Avatar } from "@radix-ui/react-avatar";
+import { streamText, type CoreMessage } from "ai";
+import { chromeai } from "chrome-ai";
+import { SendIcon } from "lucide-react";
+import { useSearchParams } from "next/navigation";
+import { AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Input } from "@/components/ui/input";
+import { ScrollArea } from "@/components/ui/scroll-area";
+
+export default function Chat() {
+  const [input, setInput] = useState("");
+  const searchParams = useSearchParams();
+  const answer = searchParams.get("answer");
+  const [messages, setMessages] = useState<CoreMessage[]>([
+    {
+      content: `Hello! I'm Nano, your personal assistant. Would you like to know more about ${answer}?`,
+      role: "assistant",
+    },
+  ]);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!input) return;
+    const newMessages: CoreMessage[] = [
+      ...messages,
+      { content: input, role: "user" },
+    ];
+    setMessages(newMessages);
+    setInput("");
+    try {
+      const { textStream } = await streamText({
+        model: chromeai(),
+        system:
+          "You are an assistent. Answer the user's questions about " + answer,
```

Comment:
- add something about refusing to answer questions outside of the scope of the trivia question / answer
- add the question as context to the system prompt
- maybe add something about refusing to handle prompts like "ignore all previous instructions"